.github/workflows/ci.yml

```diff
@@ -427,6 +427,28 @@ jobs:
         tesseract --version
         ./test_unstructured_ingest/test-ingest-dest.sh
 
+  test_ingest_help:
```

Comment:
can you add to description what this and the test-help are and why they were added?

Reply:
Wasn't sure if I was even going to add it yet. If it actually provides some benefit, ya I'll add some docs around it.